crates/typst/src/layout/flow.rs

```diff
@@ -5,18 +5,23 @@
 //! inline-level layoutable elements.
 
 use std::fmt::{self, Debug, Formatter};
+use std::num::NonZeroUsize;
+
+use typst_syntax::Span;
+use typst_utils::NonZeroExt;
```

Comment:
rust-analyzer sadly doesn't do it, but the general style is to import these from `crate::syntax` and `crate::utils` reexports instead of the raw crates (similar problem frequently happens with `typst_macros`).

Reply:
Good catch, I was going so fast with my prototype that I didn't even notice this :joy: